.github/workflows/deploy-gcp-tests.yml

```diff
@@ -375,9 +393,34 @@ jobs:
           --storage-location=us \
           --description="Created from commit ${{ env.GITHUB_SHA_SHORT }} with height ${{ env.SYNC_HEIGHT }}"
 
+  delete-instance:
+    name: Delete ${{ inputs.test_id }} instance
```

Comment:
I'd also like to block merging if this job fails, but I'm much less concerned about it, because it is a simple cleanup step.

Reply:
The only scenario where the deletion would fail is if the instance was not created because of a failure, but we anyways run the delete job just to ensure any odd scenario is being considered.

We also don't want to fail this job as we've always done, so I'm keeping the `continue-on-error: true` this step had before becoming a job.